### PR TITLE
fix: Teleport not working in portable experiences

### PR DESCRIPTION
**Pull request by sandrade-dcl**
This PR fix the bug: https://github.com/decentraland/explorer/issues/2171

### Testing
Go to https://play.decentraland.zone/branch/fix/Teleport-not-working-in-portable-experiences/index.html?ENV=org and execute the next command in the Chrome console: `spawnPortableExperienceScene("urn:decentraland:off-chain:static-portable-experiences:portal-experience", "main")`

### CODE REVIEW

File: kernel/packages/unity-interface/UnityParcelScene.ts [hunk 1]
```diff
@@ -59,7 +60,15 @@ export class UnityPortableExperienceScene extends UnityScene<LoadablePortableExp
     this.worker
       .getAPIInstance(ParcelIdentity)
       .then((parcelIdentity) => {
+        parcelIdentity.land = {
+          baseUrl: this.data.data.land!.baseUrl,
+          mappingsResponse: this.data.data.land!.mappingsResponse,
+          baseUrlBundles: this.data.data.land!.baseUrlBundles,
+          sceneId: this.data.data.land!.cid,
+          sceneJsonData: this.data.data.land!.sceneJsonData
+        }
```
Comment (menduz) [inline kernel/packages/unity-interface/UnityParcelScene.ts]: We should find another mechanism to do this, since PX are not tied to any LAND

File: kernel/packages/unity-interface/portableExperiencesUtils.ts [hunk 2]
```diff
@@ -128,7 +130,14 @@ export async function getLoadablePortableExperience(data: {
...
       contents: mappings,
-      icon: sceneJsonData.display?.favicon
+      icon: sceneJsonData.display?.favicon,
+      land: {
+        baseUrl: baseUrl,
+        baseUrlBundles: '',
+        cid: sceneUrn,
+        sceneJsonData: sceneJsonData,
+        mappingsResponse: mappingsResponse
+      }
```
Comment (menduz) [inline kernel/packages/unity-interface/portableExperiencesUtils.ts]: why is this necessary?

**Review by menduz — CHANGES_REQUESTED** [review]

File: kernel/packages/unity-interface/UnityParcelScene.ts (hunk 1 above)
Comment (sandrade-dcl) [reply kernel/packages/unity-interface/UnityParcelScene.ts]: Yeah, take a look to my latest commited changes. With the approach of creating a `isPortableExperience` flag that you suggested me, it is not already necessary to set a land for a PE parcel identity.

File: kernel/packages/unity-interface/portableExperiencesUtils.ts (hunk 2 above)
Comment (sandrade-dcl) [reply kernel/packages/unity-interface/portableExperiencesUtils.ts]: With my latest commits it is not already needed 👍 

The problem was that the `getSceneData()` function (in `RestrictedActions.ts`) needed to use `this.parcelIdentity.land.sceneJsonData` to check some conditionals. Now, with the `isPortableExperience` flag we don't need to call `getSceneData()` when the parcel identity is a PE.

Please take a look to the current changes, thanks!

**Review by menduz — APPROVED** [review]

File: kernel/packages/shared/apis/ParcelIdentity.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export interface IParcelIdentity {
 export class ParcelIdentity extends ExposableAPI implements IParcelIdentity {
   land!: ILand
   cid!: string
+  isPortableExperience!: boolean
```
Comment (menduz) [inline kernel/packages/shared/apis/ParcelIdentity.ts]:
```suggestion
  isPortableExperience: boolean = false
```

Comment (menduz) [reply kernel/packages/shared/apis/ParcelIdentity.ts]: Avoid ! whenever possible

Comment (sandrade-dcl) [reply kernel/packages/shared/apis/ParcelIdentity.ts]: Done! Thanks Agus :)
